elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base),
```

Comment:
There are a set of disabled colors for UI here:
```
ui-disabled:                          $pfelements--gray-light,
  ui-disabled--hover:                   $pfelements--gray-light,
  ui-disabled--focus:                   transparent,
  ui-disabled--text:                    $pf-color-black-600,
  ui-disabled--text--hover:             $pfelements--gray,
  ui-disabled--text--focus:             $pfelements--gray,
```

^ would any of those be valid for this?

Reply:
I think `ui-disabled` would be best. This is how `surface--base` looks on our normal `pfe-band`.
![image](https://user-images.githubusercontent.com/330256/91449258-e4139780-e848-11ea-9cc9-ad80cad10ee9.png)

If I switch from `surface--base` to `ui-disabled`, this looks better.
![image](https://user-images.githubusercontent.com/330256/91449368-05748380-e849-11ea-911e-911f5451eb19.png)